download.phtml

```diff
@@ -41,17 +41,17 @@
   <h3><?php echo T_("Sandbox")?></h3>
   <ul class="imagelist">
     <li class="imagelistitem" style="background-image: url(<?=$top_dir?>/images/icons/flatpak.png);">
-      <p class="justify"><?php printf (T_("Flatpaks are bundles, which contain beneath the program also all required libraries. If your distribution is too old, to offer all required libraries, but has the package <a href=\"%s\">Flatpak</a>, you can install a recent version of <span class=\"gnucash\">GnuCash</span> from"), $URL_FLATPAK)?></p>
+      <p class="justify"><?php printf (T_("Flatpaks are programs that bundle in all required libraries. If you have <a href=\"%s\">Flatpak</a> installed, you can grab any recent version of <span class=\"gnucash\">GnuCash</span> from"), $URL_FLATPAK)?></p>
       <p class="justify"><a href="<?php echo $URL_FLATHUB;?>"><img width='240' alt='Download on Flathub' src='https://flathub.org/assets/badges/flathub-badge-en.png'/></a></p>
       <p class="justify"><a href="<?php 
       /* Translators: %s is the base url of the gnucash wiki, adjust this,
          if a localized instance of the Flatpak page exists like "%sDe/Flatpak" */
-        printf (T_("%sFlatpak"), $URL_WIKI);?>"><?php printf (T_("Known Issues and other details …"));?></a></p>
+        printf (T_("%sFlatpak"), $URL_WIKI);?>"><?php printf (T_("Known issues and other details…"));?></a></p>
```

Comment:
"Known Issues" is a commonly-used term in software. Capitalization of the phrase is appropriate.

Reply:
I can fix it, if it helps translators, but I do not agree with the Ellipsis change.
https://en.wikipedia.org/wiki/Ellipsis#In_American_English and around.
My current style:

-  "…" omitting letters from the last word,
- " …" omitting words,
- undecided for full stop cases.

Reply:
Those usages are in the context of a quotation, but in general usage ellipses indicate that something is being left off. What's being left off here?

Reply:
You go to a new page from https://www.gnucash.org/download.phtml, so I thought to use the common "thing…" scheme.

Reply:
That's a special case for menu items in an app that open a dialog box that requires further input to complete the action. It doesn't apply to normal web links. (I almost said that it doesn't apply to web pages, but sometimes they'll run javascript with menus that do get the ellipsis.) If you want to punctuate it use a full stop, but for a link in a sidebar I don't think even that is called for.